src/uct/ib/base/ib_iface.c

```diff
@@ -80,7 +80,7 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    ucs_offsetof(uct_ib_iface_config_t, tx.min_sge), UCS_CONFIG_TYPE_UINT},
 
   {"TX_CQ_MODERATION", "64",
-   "Number of send WQEs for which completion is requested.",
+   "Number of work requests to post before requesting a send completion.",
```

Comment:
Maybe
"Maximum number of send WQEs whose completion can be reported with a single CQ element."
?

Reply:
How about "Maximum number of send WQEs which can be posted without requesting a send completion"?

Reply:
Sounds good!